### PR TITLE
Fixed changeThreadTitle and added changeThreadImage

### PR DESCRIPTION
**Pull request by kapi2289**
I fixed changeThreadTitle method because of #1545065 error and
added new methods:
`changeThreadImage(self, image_id, thread_id=None, thread_type=ThreadType.USER):`
`changeThreadImageRemote(self, image_url, thread_id=None, thread_type=ThreadType.USER):`
`changeThreadImageLocal(self, image_path, thread_id=None, thread_type=ThreadType.USER):`

### CODE REVIEW

**Comment by kapi2289** [comment]
This is only my proposition, you can edit my code like you want and add it, thank you :wink:

**Comment by madsmtm** [comment]
Of course, just wanted to give you the chance to make the changes yourself,
as per common practice ;)

**Comment by kapi2289** [comment]
Okey, I'll try to make changes but without onImageChange because I don't know how to do it. And I have one question because I'm new to GitHub. When I'll make changes do I need to open new pull request?

**Comment by madsmtm** [comment]
Nope, you just push the changes to your own branch directly :)


- Mads

File: fbchat/client.py [hunk 1]
```diff
@@ -1094,18 +1138,17 @@ def changeThreadTitle(self, title, thread_id=None, thread_type=ThreadType.USER):
...
-            data['action_type'] = 'ma-type:log-message'
-            data['log_message_data[name]'] = title
-            data['log_message_type'] = 'log:thread-name'
+            data = {
+                'thread_name': title,
+                'thread_id': thread_id,
+            }
 
-            return self._doSendRequest(data)
+            j = self._post(self.req_url.THREAD_NAME, data, fix_request=True, as_json=True)
```
[the remark this answers is not in the document]
Comment (kapi2289) [reply fbchat/client.py]: I looked what it returnes, and I think there is nothing useful.

**Comment by madsmtm** [comment]
Thanks for your contribution!

**Pull request merged**